### ISSUE TITLE
[Feature Request] upgrade to server 2022 -- deactivate internet explorer

### ISSUE THREAD

**Issue by sxinger** [Missouri-BMI/service-workbench-on-aws#4]
As @aaronmbruce found out that server 2022 will resolve the IE issue, so here is the new plan: 
1. we will upgrade to server 2022 for all new workspaces
2. for existing workspaces: 
     a. 2 outback engineering workspaces (MRKFW-PMP-Queries, EDC-RUN-SWB) -- **leave them as they are**
     b. 2 NLP workspaces (NLP-study-workspace, ALS-NLP) -- **leave them as they are**
     c. 3 standard workspaces for research project (Shraddha-Workspace, AnushkaWorkspace, Song-workspace) - I will ask my students to save their current work, terminate the workspace and re-deploy the new version once it is ready

----old notes----------------
Please investigate what are the following internet explorer shortcuts for: 
![image](https://user-images.githubusercontent.com/19035729/139316804-7c9c851c-261b-4ea2-86ba-e8af4a4d935b.png)

If they are not anything critical to running SWB workspace, please delete them from desktop.

**Comment by sxinger** [comment]
Let's make sure the new server is behaving as expected before release it into production

**Comment by sxinger** [comment]
Let's also make sure that the following steps are completed for production release: 
1. testing and validating that the new workspace type is having as expected
2. include detailed description of the new workspace (cumulative of all features carried over from old release), like 
![MicrosoftTeams-image (4)](https://user-images.githubusercontent.com/19035729/139711383-09fb4ca3-2bb2-4494-a4cf-7895bf99c8b9.png)
3. Make sure we have three server sizes configured as all other releases, like, 
![MicrosoftTeams-image (5)](https://user-images.githubusercontent.com/19035729/139711536-7ceafa30-8452-4230-b904-700b99b4bd27.png)
Or also specified in: https://nextgenbmi.atlassian.net/wiki/spaces/NBCA/pages/297598977/Service+Workbench+-+Administrator+s+Guide#Provision-Workspace-Type

**Comment by aaronmbruce** [comment]
The new v5.1 has windows 2022 and a 40gb drive

**Comment by vasanthi014** [comment]
@aaronmbruce  will move to production and close this ticket.

**Comment by aaronmbruce** [comment]
This is now in prod
